examples/timeline/styling/weekStyling.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+    <title>Timeline | Grid styling</title>
+
+    <script src="http://cdnjs.cloudflare.com/ajax/libs/moment.js/2.18.1/moment-with-locales.min.js"></script>
+    <script src="../../../dist/vis.js"></script>
+    <link href="../../../dist/vis-timeline-graph2d.min.css" rel="stylesheet" type="text/css"/>
+
+    <style type="text/css">
+        body, html {
+            font-family: sans-serif;
+        }
+
+        /* alternating column backgrounds */
+        .vis-time-axis .vis-grid.vis-odd {
+            background: #f5f5f5;
+        }
+    </style>
+
+</head>
+<body>
+
+<p>
+    Week numbers are calculated based on locales. For this to properly work, the timeline must be loaded with locales.
+    To localize the Timeline, one has to load a version of moment.js including locales. To set a locale, specify option <code>{locale: STRING}</code>.
+    To set the scale to use week numbers, use for example <code>{scale: 'week', step: 1}</code>. For example, the 'en' locale supports week number
+    calculations using US week numbers. Check out what's happening if you select another locale.
+</p>
+
+<p>
+    <label for="locale">Select a locale:</label>
+    <select id="locale">
+        <option value="en">en</option>
+        <option value="it">it</option>
+        <option value="nl">nl</option>
+        <option value="de" selected>de</option>
+    </select>
+</p>
+
+<div id="visualization"></div>
+
+<script type="text/javascript">
+    // DOM element where the Timeline will be attached
+    var container = document.getElementById('visualization');
+
+    // just a group for the effects
+    var groups = new vis.DataSet();
+    groups.add([{id: 1, content: "ISO Weeks"}, {id: 2, content: "US Weeks"}]);
+
+    // Create a DataSet (allows two way data-binding)
+    var items = new vis.DataSet([
```

Comment:
In general, I agree, but I thought about this again and think it's maybe not a good idea. First reason: We would already be relying on moment's lib code to determine week numbers to create the items - but I think to prove that things are working properly, we should not rely on something which could fail itself, thus manually writing out the week start and end seems to be fine to me. Second: I specifically wanted to display the week calculation over turn of the year, thus the for loop should always return the same result. But thanks to your comment it came to my mind to extend the examples until the first of May so that we also cover the special case when a (ISO) week start aligns with a month's start.
Nevertheless, if you still strongly vote for a loop, I will implement it.